pack/tfa/tof_encounter.json

```diff
@@ -1,164 +1,252 @@
         "faction_code": "mythos",
         "flavor": "Harlan has custody of the relic, but he believes he is being followed. The last time you heard from him, he was in his office at the university.",
         "illustrator": "Joshua Cairós",
         "name": "Harlan Is in Danger!",
         "pack_code": "tof",
         "position": 118,
         "quantity": 1,
+        "stage": 1,
         "text": "<b>Objective</b> - Only investigators in Miskatonic University can spend the requisite number of clues, as a group, to advance.",
         "type_code": "act"
     },
     {
-        "back_text": "<em class='bbc'>You search the exhibit where the relic was to be on display, looking for signs of where it may have been taken. Though you find no concrete evidence, on the ground near the exhibit entrance, you do notice several recent scuff marks made by men's shoes. You ask Drew who was last to see the exhibit after Alejandro left. \"Some suits. Government types,\" he replies. \"They had clearing to see the exhibit from Mr. Walsted, the curator.\" It's far from proof of the relic's location, but it's a lead&mdash;hopefully one that will point you in the right direction.<br/></em>Put the set-aside Town Hall location into play. If it was already in play, add add 1[per investigator] clues to it. Attach the set-aside Relic of Ages to the Town Hall. <br/>If there are 2 or fewer act decks in play, add an additional 1[per investigator] clues to the Town Hall.<br/>Advance to Act 3a-\"Find the Relic.\"",
+        "code": "04118b",
+        "encounter_code": "threads_of_fate",
+        "encounter_position": 6,
+        "faction_code": "neutral",
+        "flavor": "When you find him, Harlan is babbling incoherently and won't respond to anything you say.",
+        "health": 1,
+        "hidden": true,
+        "illustrator": "Mark Winters",
+        "is_unique": true,
+        "name": "Harlan Earnstone",
+        "pack_code": "tof",
+        "position": 118,
+        "quantity": 1,
+        "subname": "Historical Theorist",
+        "text": "<b>Revelation</b> - Put Harlan Earnstone into play in Easttown.\n[action] Discard the top 3 cards of your deck: <b>Parley.</b> You try to snap Earnstone out of his daze. Test [willpower] (4). If you succeed, place 1 clue on him <i>(from the token pool)</i>.",
+        "traits": "Bystander. Miskatonic.",
+        "type_code": "asset"
+    },
+    {
+        "back_flavor": "You search the exhibit where the relic was to be on display, looking for signs of where it may have been taken. Though you find no concrete evidence, on the ground near the exhibit entrance, you do notice several recent scuff marks made by men's shoes. You ask Drew who was last to see the exhibit after Alejandro left. \"Some suits. Government types,\" he replies. \"They had clearing to see the exhibit from Mr. Walsted, the curator.\" It's far from proof of the relic's location, but it's a lead-hopefully one that will point you in the right direction.",
```

Comment:
'but it's a lead-hopefully one' - I've been styling the long dashes used on cards as space-dash-space so: 'but it's a lead - hopefully one'. Worth changing for consistency?